src/tiled/mainwindow.cpp

```diff
@@ -316,6 +316,7 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
     connect(mUi->actionZoomOut, SIGNAL(triggered()), SLOT(zoomOut()));
     connect(mUi->actionZoomNormal, SIGNAL(triggered()), SLOT(zoomNormal()));
     connect(mUi->actionFullScreen, &QAction::toggled, this, &MainWindow::setFullScreen);
+    connect(mUi->actionZoom_With_Ctrl, SIGNAL(toggled(bool)),preferences, SLOT(setZoomPref(bool)));
```

Comment:
Coding style:
* Please remove the underscores, as done for the other actions (you can edit this name in the Design mode).
* Please prefer the new syntax for signal/slot connections (see line above).